src/store/utils.ts

```diff
@@ -86,6 +87,16 @@ export function mapResult<T, E, R>(
   };
 }
 
+export function createRemoteResult<T>(
```

Comment:
`createRemoteResult(foo, true)` is really hard to read with the boolean literal. Would you mind splitting this into two functions? Like `remoteDataLoaded(foo)` and `remoteRefreshing(foo)`.